sceptre/cli.py

```diff
@@ -580,6 +581,27 @@ def get_stack_policy(ctx, environment, stack):
     write(response.get('StackPolicyBody', {}))
 
 
+@cli.command(name="estimate-template-cost")
+@stack_options
+@click.pass_context
+@catch_exceptions
+def estimate_template_cost(ctx, environment, stack):
+    """
+    Estimates the cost of the template.
+
+    Prints a URI that provides an estimated cost based on the resources\
+    in the stack.
+    """
+    env = get_env(ctx.obj["sceptre_dir"], environment, ctx.obj["options"])
+    response = env.stacks[stack].estimate_template_cost()
+
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        success_message = "View the  estimated cost at:\n\n"
```

Comment:
Why not just have the output message `"View the estimated cost at https://example.com"`?

Reply:
I found it slightly easier to copy when it was on its own separate line. Though happy to change. C+p for me is unreasonably complicated with my setup. 😄